### ISSUE TITLE
unable to load from dynamic pages 

### ISSUE THREAD

**Issue by himadrinath** [tarekzrl/dailycards#1]
unable to load content from one page applications those sites using angular js and other framework for example test this link 

http://www.timesnownews.com/international/article/brics-china-india-narendra-modi-xi-jinping-bilateral-meet-8th-brics-summit-xiamen/85264

result found!

`{
  "icon": "http:////cloudfront.timesnownews.com/site/assets/images/icons/FavIcon_android_144x144.jpg",
  "canonical": "http://www.timesnownews.com/international/article/brics-china-india-narendra-modi-xi-jinping-bilateral-meet-8th-brics-summit-xiamen/85264",
  "site": "TimesNow",
  "title": "{{pageTitle}}",
  "keywords": "{{keywords}}",
  "date": "{{pubDate}}",
  "author": "@TimesNow",
  "description": "{{pageDescription}}",
  "image": "{{pageImage}}"
}`

**Comment by tarekzrl** [comment]
Hello there and thanks for using the package.
Now about the issue, if you viewed the source of the page, you would notice
that the result is an exact copy of the fields.
<meta name=keywords content={{keywords}}><meta name=author content=
{{author}}><meta itemprop=name content={{pageTitle}}><meta itemprop=
description content={{pageDescription}}>

Regards,
Tarek.

**Comment by himadrinath** [comment]
i have seen that. the page bind the mate data after JavaScript execution. can u do something about  that? so if the result don't meet certain condition it will reload the page again and wait for the JavaScript execution to finish and then try to read the meta data from that html.

i have tried to do that with

https://www.npmjs.com/package/nightmare

it load the url into browser and wait for finish the JavaScript execution and then return the html data. but the problem is its use lot more memory

`var Nightmare = require('nightmare');

var nightmare = Nightmare();

    nightmare.goto(url)
    .evaluate(function(){
      return document.documentElement.innerHTML;
    })
    .end()
    .then(function(value){
      /* Send this value to cheerio.load(value) */
    });

`
if u have any better solution it will be nice..

i have been testing it with many sites and i have tried many others npm packages but i must tell u that this package is the best from others in this field.

@tarekzrl Many many thanks to u.. 👍

**Comment by tarekzrl** [comment]
right, i'll have a look and get back to you if i find something. Thanks for pointing this out.
take care.
